controller.py

```diff
@@ -54,4 +56,88 @@ def run(quad, des_state):
                    k_p_theta * (des_theta - theta) + k_d_theta * (q_des - q),
                    k_p_psi * (des_psi - psi) + k_d_psi * (r_des - r)]]).T
 
+    print(F)
+    print(M)
     return F, M
+
+def run_LQR(quad, des_state):
```

Comment:
If you can, could you refactor this controller.py file a bit so it only includes control algoriuthm, and seperate out calibration parameters.
- pid() - move pid specific parameters into a seperate file.
- lqr() - could you refactor a bit so lqr specific matrix constant goes into a seperate file similar to model.params 

we should create a folder called "control" and only includes controller.py and params for now.